quiz.rb

```diff
@@ -19,7 +23,12 @@ def is_of_age? age, minimum_age
 # Remember p, print, puts don't return their arguments
 
 def okay_to_drink age, drinking_age
-  # your code goes here
+    if is_of_age? (age, drinking_age) == false then
+    "I'm sorry," + age +" isn't old enough to drink.  The minimum age is" + drinking_age
+    end
```

Comment:
this `end` will break your code. each `if....else` statement ends with a singular `end` statement